### PR TITLE
[AVFoundation] Fix some minor issues with the Xcode 8.3 bindings.

### PR DESCRIPTION
**Pull request by mandel-macaque**

### CODE REVIEW

**Comment by mandel-macaque** [comment]
Please look at the comments in https://github.com/xamarin/xamarin-macios/pull/1713 that were not done before the merge.

File: src/avfoundation.cs [hunk 1]
```diff
@@ -11026,18 +11026,33 @@ interface AVContentKeySession {
 		[Export ("renewExpiringResponseDataForContentKeyRequest:")]
 		void RenewExpiringResponseData (AVContentKeyRequest contentKeyRequest);
 
-		#region AVContentKeySession_AVContentKeyRecipients
+		#region AVContentKeySession_AVContentKeySessionPendingExpiredSessionReports
 
+		// binded because they are static and from a category.
+		[Static]
+		[Export ("pendingExpiredSessionReportsWithAppIdentifier:storageDirectoryAtURL:")]
+		NSDictionary[] PendingExpiredSessionReports (NSData appIdentifier, NSUrl storageUrl);
```
Comment (spouliot) [inline src/avfoundation.cs]: a method should have a verb/action, prefix it with `Get`

File: src/avfoundation.cs [hunk 2]
```diff
@@ -1931,6 +1931,10 @@ interface AVAudioSessionInterruptionEventArgs {
 
 		[Export ("AVAudioSessionInterruptionOptionKey")]
 		AVAudioSessionInterruptionOptions Option { get; }
+
+		[iOS (10, 3), NoMac, NoTV, NoWatch]
```
Comment (spouliot) [inline src/avfoundation.cs]: ^ sure ? that does not match xtro results

File: src/avfoundation.cs [hunk 3]
```diff
@@ -1931,6 +1931,10 @@ interface AVAudioSessionInterruptionEventArgs {
...
+		[Export ("AVAudioSessionInterruptionWasSuspendedKey")]
+		bool WasSuspended { get; }
```
Comment (spouliot) [inline src/avfoundation.cs]: _Value is a boolean NSNumber,_ ^ that `bool` won't work without `[BindAs]` (which is not in `cycle9`)

**Comment by spouliot** [comment]
I'll fix the last bit in a separate PR

**Pull request merged**